iron-list.html

```diff
@@ -209,6 +216,34 @@
       },
 
       /**
+       * When true, `iron-list` will render the items in a fixed-width grid.
+       * If true the `itemWidth` property will also need to be set
+       */
+      grid: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true
+      },
+
+      /**
+       * When `grid` is true this will define the width of each item in the
+       * grid. This plus the value of `itemOffset` will determine how many
+       * items are rendered per row
+       */
+      itemWidth: {
+        type: Number,
+      },
+
+      /**
+       * Used in conjunction with itemWidth. This will define the left and right
+       * margin of each grid item
+       */
+      itemOffset: {

```

Comment:
Great point, that would simplify the calculations going on throughout the component and would also simplify the API. I support pulling that out.